shared/actions/chat.js

```diff
@@ -547,12 +548,26 @@ function * _incomingMessage (action: IncomingMessage): SagaGenerator<any, any> {
               })
             }
           }
+
+          let messages = []
+          if (message.type === 'Attachment') {
+            const existingAttachment = yield select(_messageSelector, conversationIDKey, message.messageID)
```

Comment:
You are right, thank you! This code is not needed at all.

I must have confused myself somehow :(